### DISCUSSION TITLE
提问！BW2024不支持电脑端抢票，这个项目还能用来抢BW2024吗

### DISCUSSION THREAD

**Discussion by Gnayoah** [mikumifa/biliTickerBuy#76] [提问专区]
![屏幕截图_4-6-2024_15505_show bilibili com](https://github.com/mikumifa/biliTickerBuy/assets/79068081/974fb740-578f-41c8-8223-41c4dbed5b33)
今年BW好像不能用电脑端抢票了，不知道这个项目还能不能用来抢BW2024，不过我试了一下如果用手机端UA在电脑上是可以进入手机端抢票页面

**Comment by mikumifa** [comment]
今年的明日方舟和星穹铁道也是这个界面，然后也可以抢（手机和电脑的接口是一样的

**Reply by lllllis-02** [reply]
也就是说到时候准备开票了，接口选票里也会出选项的对嘛！俺个人还有个问题比如bw是三天，每天三种（好像是）票，程序里能识别出来然后分开设置嘛，就是比如一个人的身份买3天，另一个身份只想买一天，这种情况是不是只能开两个exe分开抢了。谢谢大佬的程序！mikumifa超绝技术力www

**Reply by mikumifa** [reply]
是的，提前配置好，抢票的内容，到时候直接抢就行了

**Reply by superwarma** [reply]
大佬，我有地方不理解这里是用手机端票的网址粘到配置网站中吗，还是说需要在电脑上装手机模拟器再把网址粘过去

**Reply by mikumifa** [reply]
网页版的会员购有网址

**Reply by superwarma** [reply]
感谢大佬，我不是很理解他说的这个（手机端UA在电脑上是可以进入手机端抢票页面）是什么意思，今年只粘贴会员购抢票网址进行配置的话可以吗，还是需要什么别的操作

**Reply by WittF** [reply]
> 感谢大佬，我不是很理解他说的这个（手机端UA在电脑上是可以进入手机端抢票页面）是什么意思，今年只粘贴会员购抢票网址进行配置的话可以吗，还是需要什么别的操作

1.UA=User-Agent
2.是，阿B不改接口就没问题
